src/components/TimelineEvent.js

```diff
@@ -2,9 +2,14 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  // Fill in your code here
-  return;
+const TimelineEvent = (props) => {
```

Comment:
Beautiful. Good use of classes. Would this be a place to use className instead of class?